Tests/NIOSSLTests/SSLCertificateTest.swift

```diff
@@ -161,6 +161,27 @@ internal func dumpToFile(data: Data, fileExtension: String = "") throws  -> Stri
     return filename
 }
 
+internal func dumpToFileWithCustomPath(data: Data, fileExtension: String = "", customPath: String = "") throws  -> String {
```

Comment:
Can we just update `makeTemporaryFile` and `dumpToFile` to have a `customPath` or am I missing something?

Reply:
Nope, you did not miss anything, we can go that route too.

Reply:
Addressed in [5d546a8](https://github.com/apple/swift-nio-ssl/pull/368/commits/5d546a8f830aab00132d33f2c210d20ed0b26713)